cmd/build/v2/servicecontext.go

```diff
@@ -0,0 +1,38 @@
+package v2
+
+import (
+	"github.com/okteto/okteto/pkg/filesystem"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/repository"
+)
+
+type serviceContextInterface interface {
+	IsCleanBuildContext() bool
+}
+
+type serviceContext struct {
+	isClean bool
+}
+
+func (sc *serviceContext) IsCleanBuildContext() bool {
+	return sc.isClean
+}
+
+func getServiceContext(buildInfo *model.BuildInfo) *serviceContext {
```

Comment:
If we are initializing, Should we name this as `newServiceContext`?

Reply:
93cf93c3273f887c4d4bdd3fbad218cfaf667240